### DISCUSSION TITLE
Ticker/slider tools

### DISCUSSION THREAD

**Discussion by FenyLabs** [DesModder/DesModder#854] [Ideas]
These are all of my current complaints about tickers and sliders, in descending order of usefulness

Currently, there is no easy way other than the console to set custom speed values for sliders. This would be useful for tickerless animations or other graphs that rely on measurements in seconds but don't want to use dt/tickers.

It's also quite annoying to save a graph with playing sliders or tickers, so it would be nice to mark sliders or the ticker to be set as playing when the graph is saved, but not in the actual graph. This could be done by overriding the saving functionality to modify the JSON, perhaps along with #310.

Tickers also run at different speeds on different displays since they use requestAnimationFrame, and this can be changed with the ?timeoutLoop query parameter. It would be nice if that was integrated, but wouldn't provide too much convenience. The graph would have to be shared with the query parameter anyway, but it might be nice to avoid an unnecessary page reload.

Lastly, and most obscurely, tickers and playing sliders can run at the same time in the same graph, it's simply disabled in UI. I can't think of a use for this other than maybe quick debugging or feature testing, but if you can maybe some people might find it useful. It does persist across saves and reloads, and is technically possible in vanilla with graph importing (thanks ronwor!). No idea why you would want to add this, just putting it here in case anyone else has one.

**Comment by sam-lb** [comment]
One more to add to this:

Starting and stopping multiple sliders at the same time. This can partially be done manually in vanilla desmos, but it'd be way easier if, say, you could bind sliders together to start/stop whenever you hit the play/pause button on any of them.
